template-html5-app/README.md

```diff
@@ -0,0 +1,251 @@
+# template-html5 - player-app
+
+In this directory, you'll discover how to develop a straightforward web application tailored for BrightSign players and deploy it onto the player. This web app boasts full functionality and is constructed similarly to any other web application. It incorporates HTML and Node.js TypeScript, alongside a simple BrightScript autorun file to configure the player to execute your app.
+
+## Step by Step: 
+1. [Set up the project](#development-prerequisites)
+2. [Set up your player](#set-up-player)
+3. [Familiarize yourself with the project, customize it if wanted](#contained-in-this-folder)
+4. [Build your code with webpack](#building-your-code)
+5. [Deploy the built code to your player](#deploying-code-to-the-player)
+6. [Run the code on your player](#restarting-the-app-on-the-player)
+
+## Development Prerequisites
+Node.js and nvm are required for installation, as specified in the README at the root of the repository.
+
+Then initialize for node:
+```sh
+nvm use 14.16.7
```

Comment:
It's a typo. Please check out the latest readme in main. There are other adjustments for order as well.

This should be updated to 14.17.6.